src/Synapse/Synapse/Models/SynapseAnalyticsManagementClient.cs

```diff
@@ -462,6 +484,54 @@ public List<RestorePoint> ListSqlPoolRestorePoints(string resourceGroupName, str
             }
         }
 
+        public void DeleteSqlPoolRestorePoint(string resourceGroupName, string workspaceName, string sqlPoolName, string sqlPoolRestorePointCreationTime)
+        {
+            try
+            {
+                if (string.IsNullOrEmpty(resourceGroupName))
+                {
+                    resourceGroupName = GetResourceGroupByWorkspaceName(workspaceName);
+                }
+
+                if (!TestSqlPoolRestorePoint(resourceGroupName, workspaceName, sqlPoolName, sqlPoolRestorePointCreationTime))
+                {
+                    throw new InvalidOperationException(string.Format(Properties.Resources.SqlPoolRestorePointDoesNotExist, sqlPoolRestorePointCreationTime));
+                }
+
+                bool isNumeric = sqlPoolRestorePointCreationTime.All(char.IsDigit);
+                if (isNumeric)
+                {
+                    this._synapseManagementClient.SqlPoolRestorePoints.Delete(resourceGroupName, workspaceName, sqlPoolName, sqlPoolRestorePointCreationTime);
+                    return;
+                }
+                this._synapseManagementClient.SqlPoolRestorePoints.Delete(resourceGroupName, workspaceName, sqlPoolName, DateTime.Parse(sqlPoolRestorePointCreationTime,
+                                          System.Globalization.CultureInfo.InvariantCulture).ToFileTimeUtc().ToString());
```

Comment:
The logic here handles both datetime and datetimeoffset. does the SQL command do the same thing?